### DISCUSSION TITLE
Entrega practica html

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#261] [Q&A]
¿Hasta que horas tenemos para entregar la práctica de html el domingo?

**Comment by Ingrzs** [comment]
Solo digo que hasta el domingo asi que serían hasta las 11:59 pm.

**Reply by GloriaCoralCerecedo** [reply]
Oh vale, gracias

**Comment by JuanSalvador73** [comment]
Si, hasta las 12 hrs medianoche del día de hoy domingo tenemos oportunidad.
